xl2times/transforms.py

```diff
@@ -3145,7 +3150,29 @@ def apply_final_fixup(
 
         df = pd.concat(df_list)
 
-    tables[Tag.fi_t] = df
+    # Clean up
+    # TODO: Do this comprehensively for all relevant tables
+    # TODO: Duplicates should only be removed if in the same file/module
+    keep_cols = [
+        "attribute",
+        "region",
+        "process",
+        "commodity",
+        "other_indexes",
+        "year",
+        "year2",
+        "timeslice",
+        "currency",
+        "limtype",
+        "sow",
+        "stage",
+        "module_name",
+    ]
+    df.dropna(subset="value", inplace=True)
+    drop_cols = [col for col in df.columns if col != "value" and col not in keep_cols]
```

Comment:
`keep_cols` should be a set so that the `not in` check is fast. We can use `list(keep_cols)` below if `drop_duplicates` requires a list, and do an O(n) conversion to a list once, instead of an O(n) list membership check for each column here.

Reply:
Thanks!